File: V.Vennela/Basics of Programming/Functions and Arrays/SpanOfArray.java
import java.io.*;
import java.util.*;

public class SpanOfArray{

public static void main(String[] args) throws Exception {
    // write your code here
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int a[] = new int[n];
    for(int i=0;i<n;i++)
    {
        a[i]=sc.nextInt();
    }
    int min=a[0],max=a[0];
    for(int i=1;i<n;i++)
    {
        if(a[i]>max)
            max=a[i];
         if(a[i]<min)
            min=a[i];
    }
    System.out.println(max-min);
 }

}

File: Meghana A S/Functions and Arrays/SpanOfArray.cpp
#include<iostream>
using namespace std;


int main(){
    //write your code here
    int n;
    cin>>n;
    int arr[n];
   
    for(int i=0;i<n;i++)
    {
        cin>>arr[i];
    }
     int min=arr[0];
    int max=arr[0];
    for(int i=0;i<n;i++)
    {
         if(arr[i]<min)
        {
            min=arr[i];
        }
        if(arr[i]>max)
        {
            max=arr[i];
        }
    }
    cout<<max-min;
    
}
